src/components/admin/UserProfileModal/components/header/ProfileModalAvatar.tsx

```diff
@@ -91,11 +87,11 @@ export const ProfileModalAvatar: React.FC<ProfileModalAvatarProps> = ({
         onChange={handleFileChange}
         accept={ACCEPTED_IMAGE_TYPES}
         ref={uploadRef}
-        className="ProfileModalAvatar__input"
+        className="ProfileModalAvatar__input invisible"
```

Comment:
Can we get rid of `ProfileModalAvatar__input`?

Reply:
as agreed, moved class name info to `data-bem`:  https://github.com/sparkletown/sparkle/pull/2931/commits/c90bfc174c869b913ac93ae9776b886e624279c8